frontend/server/src/DAO/Users.php

```diff
@@ -306,9 +306,62 @@ public static function getStatusVerified(string $email) {
                 LIMIT 1;';
         /** @var null|array{username: string, verified: bool} */
         return \OmegaUp\MySQLConnection::getInstance()->GetRow($sql);
+    }
+       /**
+       * Delete User
+       */
```

Comment:
```suggestion

    /**
    * Delete User
    */
```

that way `/` is aligned with the start of the function.

Reply:
oh okay! btw one question that linter do not fixed this automatically?

Reply:
the PHP linter is not as good :(